pkgs/tools/security/cmospwd/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, fetchurl, stdenv }:
```

One per line will keep the diff readable in the future.

```suggestion
{ lib
, fetchurl
, stdenv
):
```